src/main/java/org/kohsuke/github/GHExternalGroupPage.java

```diff
@@ -0,0 +1,34 @@
+package org.kohsuke.github;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+/**
+ * A list of external groups.
+ *
+ * @author Miguel Esteban Gutiérrez
+ */
+public class GHExternalGroupPage {
```

Comment:
I don't think this needs to be public. 
```suggestion
class GHExternalGroupPage {
```